package.json

```diff
@@ -81,10 +83,13 @@
     "foundation-sites": "^5.5.3",
     "get-env": "^0.4.0",
     "glob": "^7.0.5",
+    "history": "2.1.2",
     "http-server": "^0.9.0",
     "img-loader": "^1.3.1",
     "imports-loader": "^0.6.5",
     "ismobilejs": "^0.4.0",
+    "isomorphic-fetch": "http://registry.npmjs.org/isomorphic-fetch/-/isomorphic-fetch-2.2.1.tgz",
+    "jquery": "^3.1.1",
```

Comment:
@alexose -- Do we need jquery? I know we are trying to pull it out of the codebase if we can.